routes/deleteWarehouse/deleteWarehouse.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const deleteWarehouse = express.Router();
+const fs = require('fs');
+
+//define variables
+let warehouses = [];
+let inventories = [];
+
+//middleware to fetch data from JSON file
+deleteWarehouse.use((req, res, next) => {
+  warehouses = [];
+  inventories = [];
+  //loads new data on to the arrays
+  warehouses = JSON.parse(fs.readFileSync('./data/warehouses.json'));
+  inventories = JSON.parse(fs.readFileSync('./data/inventories.json'));
+  next();
+});
+
+//DELETE request
+deleteWarehouse.delete('/:location', (req, res, next) => {
```

Comment:
So instead of getting the warehouse name and then changing it to lowercase can we use the id of warehouse for searching the warehouse from file